cg/store/api/findbusinessdata.py

```diff
@@ -208,6 +208,38 @@ def families_by_subject_id(
                 cases.add(case)
         return cases
 
+    def get_cases_from_ticket(self, ticket_id: int) -> Query:
+        return self.Family.query.join(models.Family.links, models.FamilySample.sample).filter(
+            models.Sample.ticket_number == ticket_id
+        )
+
+    def get_customer_id_from_ticket(self, ticket_id: int) -> str:
+        """Returns the customer related to given ticket"""
+        return (
+            self.Sample.query.filter(models.Sample.ticket_number == ticket_id)
+            .first()
+            .customer.internal_id
+        )
+
+    def get_samples_from_ticket(self, ticket_id: int) -> List[models.Sample]:
+        return self.query(models.Sample).filter(models.Sample.ticket_number == ticket_id).all()
+
+    def get_samples_from_flowcell(self, flowcell_id: str) -> List[models.Sample]:

```

Comment:
Would this make sense?
```suggestion
    def get_samples_from_flowcell(self, flowcell_name: str) -> List[models.Sample]:

```